js/plugin/popups.ts

```diff
@@ -315,9 +342,16 @@ class BrowserModPopup extends LitElement {
         --padding-x: var(--popup-padding-x, 24px);
         --padding-y: var(--popup-padding-y, 20px);
       }
-
+      .content {
+        -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+        -webkit-focus-ring-color: rgba(0, 0, 0, 0);
```

Comment:
What does this do?

Reply:
Fixes #615 making sure that devices like Android don't put a focus ring around the popup.